cellengine/resources/compensation.py

```diff
@@ -75,11 +75,11 @@ def from_spill_string(spill_string: str) -> Compensation:
         channels = [arr.pop(0) for _ in range(length)]
 
         properties = {
-            "_id": None,
+            "_id": "",
             "channels": channels,
             "spillMatrix": [float(n) for n in arr],
-            "experimentId": None,
-            "name": None,
+            "experimentId": "",
+            "name": "",
```

Comment:
This change is odd actually. These aren't valid values for our API. Other possible solutions? Can the props be omitted entirely?

"name" at least can be "file-internal." CellEngine calls it "File-Internal (filename.fcs)", but you don't have access to the filename here.